geometry_msgs/Vector3 euler_angles

---

geometry_msgs/Quaternion quaternion